### ISSUE TITLE
Images

### ISSUE THREAD

**Issue by Fycbicbeth** [hungqcao/chegg-solutions-saver#6]
Images aren't downloaded, but the link to the Chegg cdn is called. Must have an internet connection to view the pictures, and if Chegg restricts access to its cdn many solutions become useless. Is it possible to have the program download the actual image files in a solution?

**Comment by hungqcao** [comment]
Yeah, that's expected. I didn't download the picture since it requires to manipulate the downloaded html. But it's possible
